pkg/controllers/restore/controller.go

```diff
@@ -126,6 +130,17 @@ func Register(
 }
 
 func (h *handler) OnRestoreChange(_ string, restore *v1.Restore) (*v1.Restore, error) {
+
+	if h.metricsServerEnabled {
+		restoreList, err := h.restores.List(k8sv1.ListOptions{})
+		if err != nil {
+			logrus.Error("Error getting Restore CR list. Failed to update metrics.")
+			return nil, err
+		}
+
+		monitoring.UpdateRestoreMetrics(restoreList)
```

Comment:
Similarly, should be run in a background loop, and suggest using backoffs for apierrors